From: Simon McVittie <smcv@debian.org>
Date: Fri, 28 Aug 2020 10:26:39 +0100
Subject: data: Remove deprecated StandardOutput=syslog

This is now equivalent to StandardOutput=journal, and recent versions
of systemd log a warning for it. Instead of updating it to journal,
remove it: the default if not specified is to respect the
DefaultStandardOutput setting from /etc/systemd/system.conf, which
in turn defaults to journal. Removing the explicit setting here means
gdm is included if a sysadmin edits system.conf to change the logging
destination for (almost) all system services at the same time, which
seems desirable.

Similarly, StandardError defaults to DefaultStandardError, which
in turn defaults to inherit, so we don't need to override it here.

Bug: https://gitlab.gnome.org/GNOME/gdm/-/issues/623
Forwarded: https://gitlab.gnome.org/GNOME/gdm/-/merge_requests/108
Signed-off-by: Simon McVittie <smcv@debian.org>
---
 data/gdm.service.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/data/gdm.service.in b/data/gdm.service.in
index fbbfb52..17e8a8d 100644
--- a/data/gdm.service.in
+++ b/data/gdm.service.in
@@ -25,8 +25,6 @@ KillMode=mixed
 Restart=always
 IgnoreSIGPIPE=no
 BusName=org.gnome.DisplayManager
-StandardOutput=syslog
-StandardError=inherit
 EnvironmentFile=-${LANG_CONFIG_FILE}
 ExecReload=/bin/kill -SIGHUP $MAINPID
 KeyringMode=shared
